Felienne/hedy
# [UI IMPROVEMENT] Remove code palette, re-structure adventure view

**Description**
In this PR we remove the code palette on the left-side of the screen. Creating more space for the adventures and code editor. The adventures are re-structured to have a structured view: combining an explanation with the corresponding example code on the same row.

**Fix for**
This PR fixes #1837.

**How to test**
Navigate to any Hedy level and notice the missing code palette. Also notice the new structure of the adventures.
